core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1121,6 +1121,10 @@ public ChangeGraphResponse changeGraph(Collection<JsonFeature> collection) {
         }
     }
 
+    public String getBlockedRectangularAreas() {
+        return blockedRectangularAreas;
```

Comment:
I need to access 'blockedRectangularAreas' in my subclass of GraphHopper. But I agree with you, this is not related to this PR. Will cancel this change.